.github/workflows/ci-only.yml

```diff
@@ -0,0 +1,32 @@
+name: ci-only
+
+on:
+  push:
+    branches: [ "*" ]
+  pull_request:
+    branches: [ master ]
```

Comment:
Why only master?

Reply:
So, should we enable it for both push and pull request?